ikenna-portfolio/portfolio/src/main/webapp/script.js

```diff
@@ -12,160 +12,76 @@
-  results are pictured above. From left to right respectively we have the original image, the generated image in the \
-  rotated domain, and the generated image in the right side up domain using the rotated domain image. <b>You can tell it\'s \
-  a generated image based on the noisiness of the image</b>",
-        
-  "Deep photo style transfer is an extension to the deep learning model Neural Style. Neural Style takes a content and style \
-  image as inputs and attempts to generate the content image in the style of the style image. One shortcoming of this model \
-  is that any generated image looks like a painting. Deep photo attempts to remedy this problem by using masks, laplacian \
-  matting, an extra loss function (see paper for more info). So far, my partner and I (this was a school project), have \
-  implemented Neural Style using the help of the tutorial found below. Doing such has yielded the result pictured above \
-  where from left to right is the content image, the style image, and the created image. Our progress on the deep photo \
-  portion of the project is technically done, but it looks terrible and was made using code we didn’t really understand (the \
-  original codebase was made in lua whereas we were used to python deep learning projects). Going forward, we hope to \
-  implement the model fully and integrate it into a web application.",
-        
-  "Although this project hasn’t been worked on for a while for reasons I’ll explain later, I still think it’s worth sharing. \
-  This project attempts to create a game engine that removes all dependencies from systems to make it multithreaded and modular. \
-  Although I technically succeeded in doing such I found out that based on the way opengl (a graphics library) renders its data, \
-  I am not allowed to render across multiple threads. The fix for this would be a synchronization at the end of each step of \
-  the the game state, but that seemed like an unrewarding solution. So, to this day, I am thinking of better solutions/trying to \
-  see how I can abstract the messaging system away from the game engine to make it easier to implement everything else. The \
-  abstraction would also allow me to implement other projects I might want multithreaded such as a genetic algorithm builder \
-  or neural net generator."
-];
-
-const links = [
-  "Here's a link to the <a href=\"https://arxiv.org/pdf/1703.10593.pdf\">paper</a>.",
-
-  "Here's a link to the <a href=\"https://arxiv.org/pdf/1703.07511.pdf\">paper</a>.<br> \
-  Here's a link to the pytorch <a href=\"https://pytorch.org/tutorials/advanced/neural_style_tutorial.html\">tutorial</a>.", 
-      
-  "Here's a link to the <a href=\"https://www.gamasutra.com/blogs/MichaelKissner/20151027/257369/Writing_a_Game_Engine_from_Scratch__Part_1_Messaging.php\">site</a> explaining the message system further."
-];
+const project_name = "<small>Project Showing: -</small>";
```

Comment:
Is it possible to move the html tag to html to only have logic for data in js?

Reply:
done